scilpy/io/utils.py

```diff
@@ -376,86 +374,235 @@ def parse_sh_basis_arg(args):
+
+
+def add_overlays_screenshot_args(parser, default_alpha=0.5,
+                                 rendering_parsing_group=None):
+    """
+    Add default arguments for overlays screenshotting.
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    default_alpha: float, optional
+        Default opacity value.
+    rendering_parsing_group: argparse.ArgumentParser object, optional
+        Group to add rendering arguments to, defaults to parser.
+    """
+    parser.add_argument("--overlays", nargs="+",
+                        help="3D Nifti image to overlay (.nii/.nii.gz).")
+
+    rpg = rendering_parsing_group or parser
+    rpg.add_argument("--overlays_as_contours", action='store_true',
+                     help="Display overlays contours and reduce the opacity "
+                          "of their inner region (see the "
+                          "`--overlays_opacity` argument).")
+    rpg.add_argument("--overlays_colors", nargs="+",
+                     type=ranged_type(int, 0, 255),
+                     default=None, metavar="R G B",
+                     help="Colors for the overlays or contours, as either a "
+                          "single color for all overlays or one for each.")
+    rpg.add_argument("--overlays_opacity", type=ranged_type(float, 0., 1.),
+                     default=default_alpha,
+                     help="Opacity value for the masks overlays. When "
+                          "combined with `--overlays_as_contours`, this will "
+                          "be the opacity of the region inside the computed "
+                          "contours. [%(default)s]")
+
+
+def add_volume_screenshot_args(parser, input_name, mandatory=True,
+                               descriptor="3D Nifti", default_cmap=None,
+                               default_alpha=1.0, cmap_parsing_group=None,
+                               opacity_parsing_group=None):
+    """
+    Add default arguments for volume screenshotting.
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    input_name: str
+        Name of the principal input volumeand argument.
+    mandatory: bool, optional
+        Whether the input argument is mandatory.
+    descriptor: str, optional
+        Descriptor for the input volume. Willbe used in help messages.
+    default_cmap: str, optional
+        Default colormap name.
+    default_alpha: float, optional
+        Default opacity value.
+    cmap_parsing_group: argparse.ArgumentParser object, optional
+        Group to add colormap arguments to, defaults to parser.
+    opacity_parsing_group: argparse.ArgumentParser object, optional
+        Group to add opacity arguments to, defaults to parser.
+    """
+    input_prefix = "" if mandatory else "--"
+    cmap_parsing_group = cmap_parsing_group or parser
+    opacity_parsing_group = opacity_parsing_group or parser
+
+    parser.add_argument(f"{input_prefix}{input_name}",
```

Comment:
I notice that you use this type of formatting everywhere. It's perfectly fine, but everywhere else in scilpy we use `"{}{}".format(input_prefix, input_name)` . Maybe we should uniformize?

Reply:
f-strings are quite new to Python, a way to shorten the uses of .format(). It's a good new standard